### DISCUSSION TITLE
How add new target HIFIONRC F7 in repository?

### DISCUSSION THREAD

**Discussion by viktor-k99** [iNavFlight/inav#8291] [General]
I created new target for HIFIONRC F7 (F7 Star mini) board. 
What I need to do for add this target to repository?

**Comment by erstec** [comment]
> I created new target for HIFIONRC F7 (F7 Star mini) board. What I need to do for add this target to repository?

https://github.com/iNavFlight/inav/blob/master/docs/policies/NEW_HARDWARE_POLICY.md
